site/html/gallery.html

```diff
@@ -25,181 +30,145 @@
+    </header>
+
+    <main>
+
+       <!-- Back to top button -->
+    <button type="button" class="btn btn-success btn-floating btn-lg" id="btn-back-to-top">
+      <i class="fa fa-arrow-up"></i>
     </button>
-  </div>
-  </div>
-  
 
-  <section class="gallery min-vh-100">
-    <h1>some glimpses of the events</h1>
-    <div class="container-lg">
-      <div class="row row-cols-1 row-cols-sm-2 row-cols-md-3">
-        <div class="col">
-          <img src="/site/images/slide1.jpg" alt="" class="galleryitems mb-4">
+      <section class="sec-container" id="CH_programming">
+        <div class="heading">
+          <h3>Programming Journey with Open Source and Future of Programming Language</h3>
         </div>
 
-        <div class="col">
-          <img src="/site/images/slide2.jpg" alt="" class="galleryitems mb-4">
-        </div>
+        <div class="box">
+          <div class="img-col">
+            <img src="/site/images/gallery/CH_programming/photo1668491797.jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668600527 (1).jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668600527.jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668600553.jpeg" alt="">
+          </div>
 
-        <div class="col">
-          <img src="/site/images/slide3.jpg" alt="" class="galleryitems mb-4">
-        </div>
+          <div class="img-col">
+            <img src="/site/images/gallery/CH_programming/photo1668600581.jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668600868.jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668600869.jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668601312.jpeg" alt="">
+          </div>
 
-        <div class="col">
-          <img src="/site/images/slide5.jpg" alt="" class="galleryitems mb-4">
+          <div class="img-col">
+            <img src="/site/images/gallery/CH_programming/photo1668601315.jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668601317.jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668601372 (1).jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668601372.jpeg" alt="">
+            <img src="/site/images/gallery/CH_programming/photo1668600581 (1).jpeg" alt="">
+          </div>
         </div>
+      </section>
 
-        <div class="col">
-          <img src="/site/images/slide5.jpg" alt="" class="galleryitems mb-4">
-        </div>
-        <div class="col">
-          <img src="/site/images/slide5.jpg" alt="" class="galleryitems mb-4">
-        </div>
-        <div class="col">
-          <img src="/site/images/slide5.jpg" alt="" class="galleryitems mb-4">
+      <section class="sec-container" id="SP_linkedlist">
+        <div class="heading">
+          <h3>Photos from Linked List Implementation in Java</h3>
```

Comment:
Event headings to h2?

Reply:
This is not the subject of concern, we can take it as a feature. This is just due to image size; showing the original width-height ratio without size reduction makes sense.
Surely heading will change.